spark/src/main/resources/interpreter-setting.json

```diff
@@ -77,6 +77,12 @@
         "propertyName": "zeppelin.spark.maxResult",
         "defaultValue": "1000",
         "description": "Max number of SparkSQL result to display."
+      },
+      "zeppelin.spark.importImplicit": {
+        "envName": "ZEPPELIN_SPARK_IMPORTIMPLICIT",
+        "propertyName": "zeppelin.spark.importImplicit",
```

Comment:
Is there any more discussion about `zeppelin.spark.importImplicit`?
If not, LGTM and i'd like to merge it into master and branch-0.6.

Thanks @sadikovi for the contribution!